db/migrate/20200824082917_create_sessions.rb

```diff
@@ -0,0 +1,29 @@
+class CreateSessions < ActiveRecord::Migration[6.0]
+  def change
+    create_table :sessions do |t|
+      t.string :session_id, null: false
+      t.string :session_type, default: ""
+      t.string :platform_name, default: ""
+      t.integer :status, default: 0
+      t.string :district_id
+      t.string :province_id
+      t.datetime :last_interaction_at
+
+      t.timestamps
+    end
+
+    ActiveRecord::Base.transaction do
+      Message.find_each do |message|
```

Comment:
@radinreth Prevent/ignore this action to run to ensure a new fresh migration won't break due to this model will be removed in the future